spec/index.bs

```diff
@@ -5587,6 +5642,41 @@ enum GPUStoreOp {
         </div>
 </dl>
 
+<div algorithm>
+    To determine if it's <dfn abstract-op>valid to draw</dfn> with {{GPURenderEncoderBase}} |encoder|
+    run the following steps:
+
+    If any of the following conditions are unsatisfied, return `false`:
+        <div class=validusage>
+            - [$Validate encoder bind groups$](|encoder|, |encoder|.{{GPURenderEncoderBase/[[pipeline]]}})
+                must be `true`.
+
+            - Let |pipelineDescriptor| be |encoder|.{{GPURenderEncoderBase/[[pipeline]]}}.{{GPURenderPipeline/[[descriptor]]}}.
+            - For each {{GPUIndex32}} |slot| `0` to
+                |pipelineDescriptor|.{{GPURenderPipelineDescriptor/vertexState}}.{{GPUVertexStateDescriptor/vertexBuffers}}.length:
```

Comment:
we need an issue or spec to say that the vertex buffers must have enough members for the draw